src/hw/connectApp.js

```diff
@@ -160,19 +167,53 @@ const cmd = ({
   appName,
   requiresDerivation,
   dependencies,
+  requireLatestFirmware,
 }: Input): Observable<ConnectAppEvent> =>
   withDevice(devicePath)((transport) =>
     Observable.create((o) => {
       const timeoutSub = of({ type: "unresponsiveDevice" })
         .pipe(delay(1000))
         .subscribe((e) => o.next(e));
 
-      const innerSub = ({ appName, dependencies }: any) =>
+      const innerSub = ({
+        appName,
+        dependencies,
+        requireLatestFirmware,
+      }: any) =>
         defer(() => from(getAppAndVersion(transport))).pipe(
           concatMap((appAndVersion): Observable<ConnectAppEvent> => {
             timeoutSub.unsubscribe();
 
             if (isDashboardName(appAndVersion.name)) {
+              // check if we meet minimum fw
+              if (requireLatestFirmware) {
```

Comment:
if user is not in dashboard in the first place, then we will never check this. is that expected? are we sure to be doing this in the correct flow?

I imagine this only work in context of also using dependencies, right? but we can't use this reliably if we just ask an app , right?

It's good for me, but please doc this limitation because that could bitten us later.

Reply:
You are correct, we need to add the same quitapp as for the dependencies, added the check on https://github.com/LedgerHQ/ledger-live-common/pull/1294/commits/649506c7bb410b35fe84cdbbae427fa300c97cc1